ui/App/DiagnosticsScreen/Json.svelte

```diff
@@ -0,0 +1,44 @@
+<!--
+ Copyright © 2021 The Radicle Upstream Contributors
+
+ This file is part of radicle-upstream, distributed under the GPLv3
+ with Radicle Linking Exception. For full terms see the included
+ LICENSE file.
+-->
+<script lang="ts">
+  export let data: unknown;
+  export let title: string | undefined = undefined;
+</script>
+
+<style>
+  .container {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    flex: 1;
+    overflow: scroll;
+  }
+
+  pre {
+    overflow: scroll;
+    font-family: var(--typeface-mono-regular);
+    font-size: 14px;
+  }
+
+  .card {
+    background-color: var(--color-foreground-level-1);
+    border-radius: 0.5rem;
+    padding: 1rem;
+    overflow: scroll;
```

Comment:
Although we’re disabling scroll bars globally we should follow best practices and use a setting that shows the scrollbar only if necessary.
```suggestion
    overflow: auto;
```

Reply:
I put the overflows there on purpose. If I remove both overflows, it ends up looking like this:

<img width="1552" alt="Screenshot 2021-11-03 at 12 18 34" src="https://user-images.githubusercontent.com/158411/140051296-e7d01f81-f875-496b-b13f-2138380a817e.png">

Reply:
:ok_hand: